monitoring/charts/grafana/templates/grafana.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: grafana
+  labels:
+    dashboards: "grafana"
+spec:
+  config:
+    log:
+      mode: "console"
+    auth:
+      disable_login_form: "false"
+    security:
+      admin_user: root
```

Comment:
if we remove these will they be the default of admin/admin? Probably better so the user doesn't need to go find it and they are equally insecure

Reply:
Leaving it out doesn't work.  Setting it to admin/admin causes it to prompt the user to change the password immediately, which in turn means the `deploy.sh` script can't deploy the dashboard successfully.

I can add the details to the readme.

Reply:
aha! gotcha, let's just add it to the readme then